### ISSUE TITLE
Ticks and text on some visual elements have poor contrast

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/my-learning-analytics#802]
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

## Expected behavior (A clear and concise description of what you expected to happen) : 

All tick marks and informational text in charts and visualizations should have a contrast ratio with the background that meets WCAG AA accessibility standard of 4.5 to 1.

## Describe the bug (Tell us what happens instead of the expected behavior) :

From the 10/28/19 accessibility review:

"Tick text as well as informational text does not have adequate contrast. ... Some users may be able to read ticks or informational text only by using user stylesheets."

## Additional context (Add any other context about the problem here) :

This issue seems to be particularly prominent in the `rangeSlider` and D3 components. The review suggests changing the tick and text color to `#000`.

**Comment by jennlove-um** [comment]
Testing passes: The ticks, lines, and text on the resources accessed and grade distribution charts now have sufficient contrast.
